### PR TITLE
Fix keywords field width on tabular view

### PR DESCRIPTION
**Pull request by migueliriano**
**Release Type:** *Breaking Changes* 

Fixes https://x-team-internal.atlassian.net/browse/XP-2470

## Description

- Added a `max-width` to `keyword` container in tabular view

## Checklist

**Before submitting a pull request,** please make sure the following is done:

- [x] set yourself as an assignee
- [x] set appropriate labels for a PR (`In Review` or `In Progress` depending on its status)
- [x] move respective Jira issue to the `IN REVIEW` column
- [x] make sure your code lints (`npm run lint`)
- [x] Flow typechecks passed (`npm run flow`)
- [x] my change requires a change to the component's documentation (`.snap`)
- [x] component's documentation (`.snap`) is changed or added accordingly
- [x] if you've fixed a bug or added code that should be tested, add unit tests
- [x] if any snapshots have been changed, verify that component still works and looks as expected and update the changed snapshot
- [x] **manually tested the component** by running it in the browser and checked nothing is broken and operates as expected!

## Related PRs

*N/A*

## Steps to Test or Reproduce
1 - Go to the Admin page and choose a tabular view 
2 - In the `keyword` field, write multiple keywords until filling the whole field.
3 - The field should keep the same width. 
 

## Impacted Areas in Application

- Admin page in tabular view the field keyword

## Screenshots

keyword_max_width

### CODE REVIEW

File: src/components/ui/SearchForm.js [hunk 1]
```diff
@@ -161,6 +161,7 @@ const tabularTheme = {
     flex: 1
     flex-shrink: 0
     min-width: 200px
+    max-width: 450px;
```
Comment (nicksp) [inline src/components/ui/SearchForm.js]: Don't you think `450px` is too much? I did some local tests and `350px` looks more than enough 😏

**Comment by nicksp** [comment]
Resolved merge conflicts.

**Comment by nicksp** [comment]
@migueliriano Is this ready for the review btw?

File: src/components/ui/SearchForm.js (hunk 1 above)
Comment (migueliriano) [reply src/components/ui/SearchForm.js]: I did some local changes to and looks like `450px` was the perfect fit for this, maybe the difference is your resolution.

@nicksp - Can you test it with this px?

**Comment by migueliriano** [comment]
@nicksp - Yes, now is ready for review, I was fixing the conflict before add the labels.

File: src/components/ui/SearchForm.js (hunk 1 above)
Comment (nicksp) [reply src/components/ui/SearchForm.js]: Yeah, I tested and it looked too big. `350px`-`370px` doesn't seem to expand search form that much and make it a bit more compact.

**Pull request merged**